packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/ResearchProjectForm/ResearchProjectForm.tsx

```diff
@@ -129,13 +144,22 @@ export default function ResearchProjectForm(props: ResearchProjectFormProps) {
         <ManagedACTAccessRequirementFormWikiWrapper
           managedACTAccessRequirementId={String(managedACTAccessRequirement.id)}
         >
-          <form onSubmit={handleSubmit}>
-            <Typography variant={'body1'}>
-              Please tell us about your project.
-            </Typography>
+          <Box
+            component={'form'}
+            onSubmit={handleSubmit}
+            sx={{
+              '& .MuiTextField-root': {
+                marginBottom: '20px',
+              },
+            }}
+          >
             <TextField
               id={'project-lead'}
-              label={'Project Lead'}
+              label={'First and last names of your project lead or PI'}
```

Comment:
updated non ADKP-specific labels to match design: https://www.figma.com/design/jaE5E1p8qv4lZ46iqVdh7y/AD-Knowledge-Portal?node-id=2725-2333&node-type=frame&m=dev